README.md

```diff
@@ -13,8 +14,51 @@ LSST DM SQuaRE api.lsst.codes-compliant microservice wrapper.  TODO
 
 * `/`: returns `OK` (used by Google Container Engine Ingress healthcheck)
```

Comment:
It's a good idea to always prefix the endpoint with the method (I know you do this for `/ccutter/<projecttype`, but I'd like to see it for all. E.g.:

`GET /`: return `'OK'` [..]